tests/test_units/test_aggregated_surfaces.py

```diff
@@ -29,11 +30,11 @@ def test_regsurf_aggregated(fmurun_w_casemetadata, rmsglobalconfig, regsurf):
     for i in range(10):
         use_regsurf = regsurf.copy()
         use_regsurf.values += float(i)
-        expfile = edata.export(use_regsurf, name="mymap_" + str(i))
+        expfile = edata.export(use_regsurf, name="mymap_" + str(i), realization=i)
```

Comment:
Is there a risk that "mymap_" can make it out of fmu-dataio and suddenly appear in filenames, metadata, etc?